src/utils/HipaceProfilerWrapper.H

```diff
@@ -20,11 +20,14 @@
 #include <AMReX_BLProfiler.H>
 #include <AMReX_GpuDevice.H>
 
+/** Whether to call amrex::Gpu::synchronize() around all profiler region */
+inline int m_do_device_synchronize = 0;
```

Comment:
Could we avoid changing things in the profiler wrapper in this PR?

Reply:
#778